File: src/extension/src/api/templates/Web/Features/Azure.Cosmos.SQL/server/sql/sqlClient.js
const CosmosClient = require("@azure/cosmos").CosmosClient;
require("dotenv").config();

// Connects the back-end to the Cosmos Core SQL Database (https://docs.microsoft.com/en-us/azure/cosmos-db/create-sql-api-nodejs)
module.exports = class SQLClient {
  constructor(databaseId, containerId) {
    this.client = new CosmosClient({
      endpoint: process.env.COSMOSDB_URI,
      auth: {
        masterKey: process.env.COSMOSDB_PRIMARY_KEY
      }
    });

    this.databaseId = databaseId;
    this.containerId = containerId;

    this.database = null;
    this.container = null;
  }

  async connect() {
    const dbResponse = await this.client.databases.createIfNotExists({
      id: this.databaseId
    });
    this.database = dbResponse.database;

    const containerResponse = await this.database.containers.createIfNotExists({
      id: this.containerId
    });
    this.container = containerResponse.container;
  }
};

File: templates/Web/_composition/Flask/Feature.Flask.Azure.Cosmos.SQL/server/sql/sqlClient.py
import azure.cosmos.cosmos_client as cosmos_client
from constants import CONSTANTS
from .settings import *


class SQLObj():
    def __init__(self):
        self.client = cosmos_client.CosmosClient(
            url_connection = connection_key,
            auth = {'masterKey': master_key}
        )
        self.db = self.client.CreateDatabase(
            {'id': CONSTANTS['COSMOS']['DATABASE']}
        )
        self.container = self.client.CreateContainer(
            self.db['_self'],
            {
                'id': CONSTANTS['COSMOS']['CONTAINER']
            }, 
            {
                'offerThroughput': 400
            }
        )

    def get_client(self):
        return self.client

    def get_db(self):
        return self.db

    def get_container(self):
        return self.container